docs/simple_switch.md

```diff
@@ -1031,6 +1031,57 @@ options:
   time synchronization.
 
 
+### BMv2 idle timeout implementation notes
+
+The BMv2 v1model implementation supports the table property
+`support_timeout`.  If not specified for a table, its value is false.
+If you assign this table property the value of true when defining a
+table, as shown in the table definition below, then BMv2 will maintain
+independent state for each entry of the table to detect when it has
+been longer than a configured time interval since the entry was last
+matched.
+
+```
+    // Definitions of actions have been omitted in this code snippet.
+    // Their definitions do not affect the behavior of the
+    // `support_timeout` table property.
+
+    table mac_da_fwd {
+        key = {
+            hdr.ethernet.dstAddr: exact;
+        }
+        actions = {
+            set_port;
+            my_drop;
+        }
+        support_timeout = true;
+        default_action = my_drop;
+    }
+```
+
+See Sections
+["Idle-timeout"](https://p4.org/p4-spec/p4runtime/v1.3.0/P4Runtime-Spec.html#sec-idle-timeout)
+and ["Table Idle Timeout
+Notification"](https://p4.org/p4-spec/p4runtime/v1.3.0/P4Runtime-Spec.html#sec-table-idle-timeout-notification)
+in the P4Runtime Specification for how a controller can configure the
+time interval for each entry it adds to such a table, and the contents
+of IdleTimeoutNotification messages sent from the switch to a
+P4Runtime controller when a table entry has not been matched for
+longer than its configured `idle_timeout_ns` duration.
```

Comment:
I wonder if this doc should make it clear that only simple_switch_grpc exposes a P4Runtime API?

Reply:
I have added an explicit mention that only simple_switch_grpc implements the P4Runtime API, not simple_switch.